chartfx-dataset/src/main/java/de/gsi/dataset/spi/AbstractDataSet.java

```diff
@@ -378,6 +377,11 @@ protected boolean equalValues(final DataSet other, final double epsilon) {
         return true;
     }
 
+    @Override
```

Comment:
JavaFX dependence in non-UI module. The solution could e.g. look something like

```java
   // in the header
   private boolean isVisible = true;
   // [..]
    public boolean isVisible() { return isVisible; };

    public void setVisible(boolean visible) {
        if (visible != isVisible) {
            isVisible = visible;
            fireInvalidated(new UpdatedMetaDataEvent(this, "removed style"));
        }
    }
```
What do you think?